### ISSUE TITLE
Suggestion: DD calculator material sorting

### ISSUE THREAD

**Issue by Syhnd** [tyejae/msf.gg.public#351]
At every shop refresh I like to know which material I need to upgrade my characters for DD.
Searching for each material is a pain in the a**

A way to sort materials would be nice.  For example grouping uniques together, "mini uniques" together, "micro uniques" together, catalysts together...

This is an add up to the #345 suggestion, the abilty to see only the required material is a must

**Comment by tyejae** [comment]
Yeah the hard part of this is that Marvel Strike Force has no concept of mini uniques, micro uniques or any unique naming. This would result in a lot of hard coding. I'll have to think if there is a way to implement this.

**Comment by Syhnd** [comment]
I saw that they were kinda sorted in the inventory page (https://msf.gg/inventory/) I thought that you could do something similar
